bb_wrapper/wrapper/bb.py

```diff
@@ -29,6 +29,7 @@ def __init__(
         gw_app_key=None,
         verify_https=True,
         cert=None,
+        **kwargs,
```

Comment:
Pode ser interessante colocar o `*args` antes dos atributos declarados para forçar o init com kwarg nos wrappers.

Vai facilitar bastante o entendimento da passagem dos parâmetros!

Reply:
Feito